kolibri/plugins/html5_app_renderer/assets/src/views/Html5AppRendererIndex.vue

```diff
@@ -69,6 +71,31 @@
       }
       this.$emit('stopTracking');
     },
+    methods: {
+      iframeMessageReceived(event) {
+        if (!event) {
+          return;
+        }
+        var message = JSON.parse(event.data);
+        if (message.action === 'stateUpdated') {
+          this.$emit('updateContentState', message.params);
+        } else if (message.action === 'hashiInitialized') {
+          var iframe = document.getElementsByTagName('iframe')[0].contentWindow;
```

Comment:
Let's scope this by instead adding a ref to the iframe in the template `ref="iframe"` and then we can do
```js
const iframe = this.$refs.iframe.contentWindow;
```
https://vuejs.org/v2/api/#ref

Reply:
Ah, nice! I think @rtibbles was trying to explain this to me during a previous review but I got caught up in other changes and forgot to double-back to this one.